packages/apostrophe/modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -25,14 +25,7 @@ module.exports = ({
   const mode = process.env.NODE_ENV || 'development';
   const pnpmModulePath = apos.isPnpm ? [ path.join(apos.selfDir, '../') ] : [];
   const dev = (mode === 'development');
-  // The default behavior of the security-headers module
```

Comment:
I was able to get rid of all this blinking shit because the real underlying problem was always that we weren't concatenating sourcemaps correctly. In my tests devtool: 'source-map' works fine now. And it doesn't use eval so it should be CSP OK.